assets/src/__tests__/d3/createHistogram.test.js

```diff
@@ -1,11 +1,21 @@
 /* global describe, test, expect */
 import createHistogram from '../../components/d3/createHistogram'
-import { histogramData } from '../testData/d3TestData'
+import {histogramData, histogramDataMoreThan100Percent} from '../testData/d3TestData'
 
 describe('createHistogram', () => {
   test('should build a histogram', () => {
     const div = document.createElement('div')
     createHistogram({ data: histogramData, width: 1000, height: 500, el: div })
     expect(div).toMatchSnapshot()
   })
+  test('should build a histogram bar chart with MyGrade line', () => {
+    const div = document.createElement('div')
+    createHistogram({ data: histogramData, width: 1000, height: 500,  el: div, myGrade:79.8})
+    expect(div).toMatchSnapshot()
+  })
+  test('should build a histogram bar chart with MyGrade line and distribution more than 100', () => {
+    const div = document.createElement('div')
+    createHistogram({ data: histogramDataMoreThan100Percent, width: 1000, height: 500,  el: div, myGrade:56.8, maxGrade: 110})
+    expect(div).toMatchSnapshot()
+  })
```

Comment:
Just added a few tests with various cases for grades distribution. please check it. 
A question do we need to add tests so that both the side grid table and distribution is loaded as part of the view. Seems to be a reasonable test. But the table is not a d3 element and the `Table` component is at Container level. so how this will be handled?

Reply:
Yes, I think for the React tests we'll be using [Enzyme](https://airbnb.io/enzyme/) and Jest. Soo and I are still exploring the best way to test containers. She refactored the API calls made from container components into a separate folder as part of this PR: https://github.com/tl-its-umich-edu/my-learning-analytics/pull/461

We'll be using Jest to mock API calls and then asserting that the React components have the correct output.